soepy/python/solve/solve_auxiliary.py

```diff
@@ -67,27 +66,23 @@ def pyth_create_state_space(model_params):
 
     Parameters
     ----------
-    model_params.num_periods : int
-        Number of periods in the state space.
-    model_params.educ_range : int
-        Range of initial condition years of education in the (simulated) sample.
-    NUM_CHOICES : int
-        Number of choices agents have in each period.
-    educ_min : int
-        Minimum number of years of education in the simulated sample.
+    model_params: namedtuple
+        Namedtuple containing all information relevant for running a simulation.
+        Includes parameters, dimensions, information on initial conditions, etc.
 
     Returns
     -------
     states : np.ndarray
-        Array with shape (num_states, 6) containing period, experience in OCCUPATION A,
-        experience in OCCUPATION B, years of schooling, the lagged choice and the type
-        of the agent.
+        Array with shape (num_states, 5) containing period, years of schooling,
+        the lagged choice, the years of experience in part-time, and the
+        years of experience in full-time employment.
     indexer : np.ndarray
         A matrix where each dimension represents a characteristic of the state space.
         Switching from one state is possible via incrementing appropriate indices by 1.
 
     Examples
     --------
+    >>> from collections import namedtuple
```

Comment:
Are we automatically running `doctest` for these tests somwhere?

Reply:
So far, no. Is there a project where you are doing it and I can see how to implement it?